src/solver.cpp

```diff
@@ -292,6 +292,7 @@ inline SolverRegistrar::SolverRegistrar(IdRegistryData& registry)
              Primitive::Fusion,
              solver::fusion::ConvOclDirectFwdFused{}.SolverDbId(),
              miopenConvolutionAlgoDirect);
+    Register(registry, ++id, Primitive::Fusion, solver::fusion::CKGEMMActiv{}.SolverDbId());
```

Comment:
This should only be added to the end of the file or it would corrupt existing serialized solutions.